packages/salesforcedx-vscode-apex/test/vscode-integration/views/testOutlineNamespace.test.ts

```diff
@@ -0,0 +1,259 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { expect } from 'chai';
+import * as events from 'events';
+import * as fs from 'fs';
+import { sandbox, SinonSandbox, SinonStub } from 'sinon';
+import * as vscode from 'vscode';
+import { APEX_GROUP_RANGE } from '../../../src/constants';
+import { ApexTestMethod } from '../../../src/views/lspConverter';
+import {
+  ApexTestGroupNode,
+  ApexTestNode,
+  ApexTestOutlineProvider
+} from '../../../src/views/testOutlineProvider';
+import { ApexTestRunner } from '../../../src/views/testRunner';
+import { generateApexTestMethod } from './testDataUtil';
+import { summaryMultipleFiles, summaryOneFile } from './testJSONOutputs';
+import {
+  testResultsMultipleFiles,
+  testResultsOneFile
+} from './testNamespacedOutputs';
+
+const jsonOneNSFilePass = {
+  summary: summaryOneFile,
+  tests: testResultsOneFile
+};
+
+const jsonMultipleNSFiles = {
+  summary: summaryMultipleFiles,
+  tests: testResultsMultipleFiles
+};
+
+describe('Test View with namespace', () => {
+  let testOutline: ApexTestOutlineProvider;
+  const apexNamespacedTestInfo: ApexTestMethod[] = generateApexTestMethod(
+    'tester'
+  );
+
+  describe('Get Tests and Create Tree', () => {
+    it('Should create one test and one class when using namespace', () => {
+      testOutline = new ApexTestOutlineProvider(
+        apexNamespacedTestInfo.slice(0, 1)
+      );
+      if (testOutline.getHead()) {
+        expect(testOutline.getHead().name).to.equal('ApexTests');
+        expect(testOutline.getHead().children.length).to.equal(1);
+        const testChildGroup = testOutline.getHead().children[0];
+        expect(testChildGroup).instanceof(ApexTestGroupNode);
+        const groupLocation = new vscode.Location(
+          apexNamespacedTestInfo[0].location.uri,
+          APEX_GROUP_RANGE
+        );
+        expect(testChildGroup.location).to.deep.equal(groupLocation);
+        expect(testChildGroup.name).to.equal(
+          apexNamespacedTestInfo[0].definingType
```

Comment:
Does `definingType` include the `tester` namespace?

Reply:
Ah i see it https://github.com/forcedotcom/salesforcedx-vscode/pull/1731/files#diff-c258f087feaf15a9cbc71d6b5c423ad2R16